script.php

```diff
@@ -404,6 +404,10 @@ public function __construct()
 			'/language/de-DE/de-DE.tpl_protostar.sys.ini',
 			'/language/de-DE/de-DE.xml',
 			// Old files from Joomla 4 language packs
+			'/administrator/language/de-DE/plg_captcha_recaptcha.ini',
+			'/administrator/language/de-DE/plg_captcha_recaptcha.sys.ini',
+			'/administrator/language/de-DE/plg_captcha_recaptcha_invisible.ini',
+			'/administrator/language/de-DE/plg_captcha_recaptcha_invisible.sys.ini',
```

Comment:
Das `recaptcha_invisible` wird ja nicht deinstalliert, sondern nur entsperrt und bleibt weiterhin installiert.
_Problem:_ Da wir es ja nicht mitbekommen, ob im nachhinein das Plugin (manuell) deinstalliert wird, würden bei einer Deinstallation die deutschen Übersetzungen weiterhin verbleiben.

Passt es dann, dass das Plugin in J5 nur noch in englischer Sprache vorhanden ist - es wird ja auch nicht mehr vom Core mit "Updates" versorgt!?

Reply:
Ok, aber die 4 ini-files wurden doch gelöscht im PR: https://github.com/joomla/joomla-cms/pull/41530/files#diff-cbe3a16b61f80e0af9df2e3c9b0a5b5689af5b3e54e66c1909102fd84db42193

Reply:
Ja das betrifft nur die Neuinstallation

Reply:
In Crowdin wird das dann auch automatisch gelöscht sein. 

Was meinst du @zero-24 ?
Wir wollen wir das lösen?